src/scripts/main.js

```diff
@@ -355,6 +356,35 @@ const people = [
 ];
 
 // eslint-disable-next-line no-console
-console.log(people); // you can remove it
 
-// write your code here
+const table = document.querySelector('.dashboard');
+
+people.forEach((person) => {
```

Comment:
you can use object destructing here
```suggestion
people.forEach(({ sex, born, died, name }) => {
```